python/core/auto_generated/qgsvectorlayer.sip.in

```diff
@@ -1042,13 +1042,15 @@ if the geometry type of the new data source matches the current geometry type of
     virtual QString loadDefaultStyle( bool &resultFlag /Out/ ) ${SIP_FINAL};
 
 
-    QgsVectorLayerFeatureCounter *countSymbolFeatures();
+    QgsVectorLayerFeatureCounter *countSymbolFeatures( bool withFids = false );
 %Docstring
 Count features for symbols.
 The method will return the feature counter task. You will need to
 connect to the symbolFeatureCountMapChanged() signal to be
 notified when the freshly updated feature counts are ready.
 
+:param withFids: If ``True`` will gather the Fids per symbol, otherwise only the count. Default ``False``.
```

Comment:
While fids is the term used in the api often, I think in doc strings we should be more verbose.

```suggestion
:param withFids: If ``True`` will gather the feature ids (fids) per symbol, otherwise only the count. Default ``False``.
```

Reply:
Should be done